### PR TITLE
Alt Coins

### PR DESCRIPTION
**Pull request by ErebusBat**
This PR makes the required changes for the transition from Bitcoin to more general Cryptocurrency.

It also:
- Updates bundler to v2 to match fetlife-web
- Normalizes all translation files
  - It is not ideal that it is not numerically sorting :/
  - But using the i18n tasks will make things easier in the long run
- You may want to compare [after the normalization](https://github.com/fetlife/translations/compare/e1f10b839b531ca78c6373c4a566a20b4637d595..alt-coins-1) to more easily see the actual changes.

Part of https://github.com/fetlife/fetlife-web/issues/15617

### CODE REVIEW

File: locales/bg.yml [hunk 1]
```diff
@@ -483,15 +483,15 @@ bg:
   support:
     benefits:
       item_1: A sexy <span>I Support FetLife</span> badge on your profile.
+      item_10: Get early <span>access to new features</span>!
```
Comment (nielsenramon) [inline locales/bg.yml:486]: Yeah not a fan of this moving here. Nothing we can do about it?

Comment (nenadjelovac) [reply locales/bg.yml:486]: @ErebusBat wait, are we trying to reorder them in the UI? As I don't think this would do it.

We could reorder it in https://github.com/fetlife/fetlife-web/blob/master/app/views/ecosystem/component/_support__benefits.html.slim 🤷‍♂️  Wdyt @ErebusBat @nielsenramon?

Comment (nielsenramon) [reply locales/bg.yml:486]: @nenadjelovac We don't want to reorder them, I think it's the command @ErebusBat ran that reordered them in the translations.

Comment (ErebusBat) [reply locales/bg.yml:486]: > are we trying to reorder them in the UI? As I don't think this would do it.
..
> Nothing we can do about it?...I think it's the command @ErebusBat ran that reordered them in the translations.

Correct...  

There are built in tasks that help in management of the files (see https://github.com/fetlife/translations/blob/master/CONTRIBUTING.md)

Running `bundle exec i18n-tasks normalize` makes sure that all the keys are in the same order in all the files.

Why is that important?

Well because when doing changes like this the workflow is something like:

1. Remove all the entries we want to chantge from the file:
    - In this case `support.step1.coingate.title`
    - `bundle exec i18n-tasks data -f yaml | bundle exec i18n-tasks tree-filter support.step1.coingate.title | bundle exec i18n-tasks data-remove`
2. Add the `support.step1.coingate.title` back to the `en.yml` file.
3. Use i18n to add it back to all other files:
    - `bundle exec i18n-tasks add-missing`

So we _can_ reorder the entries in the files... however if/when we use our tools then they can change order unpredictably which makes reviews / changes more difficult to track and leads to the possibility of the files being out of sync.

In short:  It sucks, but it we want it to be the "correct" way then we will be fighting our tools.

/cc @nielsenramon @nenadjelovac

Comment (nielsenramon) [reply locales/bg.yml:486]: @ErebusBat Then I'm good with this for now 🤷 Let's get it out 👍

**Review by nielsenramon — APPROVED** [review]
